src/wp-includes/class-wp-theme-json.php

```diff
@@ -2696,7 +2696,9 @@ private static function update_separator_declarations( $declarations ) {
 	 * @param array $options {
 	 *     Optional. An array of options for now used for internal purposes only (may change without notice).
 	 *
-	 *     @type bool   $include_block_style_variations  Includes nodes for block style variations. Default false.
+	 *     @type bool $include_block_style_variations Includes nodes for block style variations. Default false.
+	 *     @type bool $include_node_paths_only        Includes node path for block nodes. Default false.
+	 *     @type bool $include_block_elements         Includes block elements for block nodes. Default true.
```

Comment:
Based on @andrewserong's feedback, this parameter might make sense. However, I'm concerned that we would have to give this a default of `true` which maybe is a bit confusing?

Since this method is by no means specific to background images, I'd prefer if we omitted this parameter, in favor of a cleaner API, and since the performance overhead of also considering elements for only the paths is negligible.

Reply:
Make sense, Update in https://github.com/WordPress/wordpress-develop/pull/7486/commits/867ae4a9aedefb2d9c7d96769ccb5c85a134c869